test/JDBC/src/main/java/com/sqlsamples/Config.java

```diff
@@ -109,28 +94,21 @@ static String createPostgreSQLConnectionString(String URL, String port, String d
     
     private static String constructConnectionString(String prefix) {
```

Comment:
@congoamz deleted your comment by mistake. I agree we can remove the prefix argument. Will address this in my revision